snuba/web/delete_query.py

```diff
@@ -9,11 +9,12 @@
 from snuba.query.dsl import column, equals, in_cond, literal, literals_tuple
 from snuba.query.expressions import Expression
 from snuba.reader import Result
+from snuba.state import get_config
 
 
-def construct_condition(body: Dict[str, Any]) -> Expression:
+def construct_condition(columns: Dict[str, Any]) -> Expression:
```

Comment:
I think we should change this to _construct_condition since it seems like a private method used solely for deletes